src/libstd/io/mod.rs

```diff
@@ -284,6 +284,8 @@ pub use self::stdio::{stderr, stdin, stdout, Stderr, Stdin, Stdout};
 pub use self::stdio::{StderrLock, StdinLock, StdoutLock};
 #[unstable(feature = "print_internals", issue = "none")]
 pub use self::stdio::{_eprint, _print};
+#[unstable(feature = "input", issue = "none")]
+pub use self::stdio::{input, prompt};
```

Comment:
Since you put this as a separate feature below, these should be separated.
```suggestion
#[unstable(feature = "input", issue = "none")]
pub use self::stdio::input;
#[unstable(feature = "prompt", issue = "none")]
pub use self::stdio::prompt;
```
I also think it should be sorted, I am just showing this as a suggestion.